src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeCallableAttribute.cs

```diff
@@ -17,12 +23,12 @@ public NativeCallableAttribute()
         }
 
         /// <summary>
-        /// Optional. If omitted, compiler will choose one for you.
+        /// Optional. If omitted, the runtime will choose one for you based on the platform.
         /// </summary>
         public CallingConvention CallingConvention;
 
         /// <summary>
-        /// Optional. If omitted, then the method is native callable, but no EAT is emitted.
+        /// Optional. If omitted, then the method is native callable, but no export is emitted during AOT compilation.
```

Comment:
```suggestion
        /// Optional. If omitted, then the method is native callable, but no export is emitted during compilation.
```
This is not specific to AOT